src/test/ui/test-attrs/test-fn-signature-verification-for-explicit-return-type.rs

```diff
@@ -1,5 +1,5 @@
 // run-pass
-// ignore-fuchsia Test must be run out-of-process
+// needs-unwind (#103261)
```

Comment:
This turned out to be a duplicate of https://github.com/rust-lang/rust/issues/73509